packages/templates/src/raw/all-about-cars/metaData.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __, _x } from '@web-stories-wp/i18n';
+
+export default {
+  slug: 'all-about-cars',
+  creationDate: '2021-09-01T00:00:00.000Z',
+  title: _x('All About Cars', 'template name', 'web-stories'),
+  tags: [
+    _x('Technology', 'template keyword', 'web-stories'),
+    _x('Cars', 'template keyword', 'web-stories'),
+    _x('Features', 'template keyword', 'web-stories'),
+    _x('Futuristic', 'template keyword', 'web-stories'),
+    _x('White', 'template keyword', 'web-stories'),
+  ],
+  colors: [
+    { label: _x('Neon Aqua Blue', 'color', 'web-stories'), color: '#00f0ff' },
+    { label: _x('Catalina Blue', 'color', 'web-stories'), color: '#002e73' },
+    { label: _x('Light Aqua Blue', 'color', 'web-stories'), color: '#a2f6fc' },
+    { label: _x('White', 'color', 'web-stories'), color: '#fff' },
+    { label: _x('Celestial Blue', 'color', 'web-stories'), color: '#152132' },
```

Comment:
Neon Aqua Blue and Light Aqua Blue seem veery similar. Should we perhaps just keep one?

Reply:
I think we can keep both, as Light Aqua Blue is heavily used for the text and the Neon Aqua Blue is used for the patterns. and there are only five colours so not too crowding either.

The palette as seen on the details modal:
![Screenshot 2021-09-02 at 3 06 23 PM](https://user-images.githubusercontent.com/6906779/131821031-8e9c2940-ddda-4983-bad8-550d34ba6812.png)

Screenshot of the first story page:
![Screenshot 2021-09-02 at 3 06 30 PM](https://user-images.githubusercontent.com/6906779/131821039-aae1317c-d4b7-4099-846f-22683cf9c1c4.png)

And if we need to remove one, I think we can remove the Neon Aqua Blue as it is used for the patterns ( SVG ) where users don't have any control. suggest.